### ISSUE TITLE
Demos suffer from FOUC

### ISSUE THREAD

**Issue by stevesims** [Polymer/polymer-element-catalog#85]
On Safari, and possibly some other browsers too, the demos all suffer from Flash Of Unstyled Content.  This is especially visible when visiting the dialog demos page.

The resolution for this issue was documented in the Polymer docs two years ago, saying that to prevent this from happening the body tag should get an `unresolved` attribute.  The demos clearly lack this attribute.

**Comment by mbleigh** [comment]
The `unresolved` attribute is no longer a part of Polymer, but agree that something can/should be done about FOUC in demos.

**Comment by stevesims** [comment]
Yes - as it stands it creates a bad impression and makes adopting Polymer look very unattractive, and virtually impossible to sell through to management wanting to make sites that will work on iOS.

It's a shame that the `unresolved` attribute has been dropped, and curious that adding it still works (in that it makes the content be hidden) on the demo pages.

**Comment by kevinpschaaf** [comment]
Support for unresolved on body was just re-added in 1.0, so we'll be updating demos and docs to include that as a best practice going forward.

**Comment by tjsavage** [comment]
This will need fixing in each individual element demo, but leaving this here as tracking.

**Comment by ebidel** [comment]
@tjsavage have we made any progress on all the demos?
